README.md

```diff
@@ -89,11 +89,17 @@ gulp.task('sass', function () {
 
 ```
 
-To import images, configure your application to reference or copy the icons assets.
+### Import images and icons
 
-Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+In order to import GOV.UK Frontend images and icons to your project, you should configure your application to reference or copy the relevant GOV.UK Frontend assets.
+
+You can do this with one of the following:
```

Comment:
This isn't quite right – option 1 modifies the CSS to point to a different folder, but doesn't help with actually getting the images into a public facing directory. The use of 'one of the following' suggests that they should do either 1 _or_ 2 but it's not that simple.